### PR TITLE
Add Volume computing methods.

### PR DESCRIPTION
**Pull request by ErikQQY**
Signed-off-by: ErikQQY <2283984853@qq.com>

### CODE REVIEW

**Review by arubhardwaj — DISMISSED** [review]

**Comment by ErikQQY** [comment]
It looks like there are some errors, I will fix it

**Comment by ErikQQY** [comment]
@arubhardwaj Please review again.

**Review by Azzaare — APPROVED** [review]
Things look good for this PR. There some conflicts (due probably to other PR having been merged in between), but I should be able to solve that easily.

**Review by Azzaare — DISMISSED** [review]

**Comment by ErikQQY** [comment]
Sorry for the merge conflicts, I will be more careful in my future PR

**Comment by ErikQQY** [comment]
Looks like there are some errors, I will fix them.

**Comment by Azzaare** [comment]
It is fine, the conflicts were coming from the fact to merge other PR into main. Ideally, we should have few conflicts if we merge PR as soon as possible, but I was lacking time to check those recently.

Also, @ErikQQY, it seems there are some errors in the test (that comes from some volume functions). Can you update it? (you can usually test locally with `] test TheAlgorithms` or just `] test` if you are in the local environment of TheAlgorithms

**Comment by ErikQQY** [comment]
Thanks a lot!! I will fix them as soon as possible.

**Comment by ErikQQY** [comment]
@Azzaare I think it is working now, sorry for the previous errors

**Comment by ErikQQY** [comment]
The codecov failed, do I need to rewrite the algorithms to fix them?

**Comment by Azzaare** [comment]
So the failure is about the decrease of code coverage. So the code is not failing per se, but some part of the code is not covered.

If you can modify the tests to cover every case that would be great. If for some reason some parts cannot be covered, let us know and we will merge it.

**Comment by Azzaare** [comment]
So the failure is about the decrease of code coverage. So the code is not failing per se, but some part of the code is not covered.

If you can modify the tests to cover every case that would be great. If for some reason some parts cannot be covered, let us know and we will merge it.

**Comment by ErikQQY** [comment]
I think it is the error throwing section cause the codecov error

```julia
if radius < 0
    throw(DomainError)
```

I will add these error throw tests

**Comment by ErikQQY** [comment]
I think it is the error throwing section cause the codecov error

```julia
if radius < 0
    throw(DomainError)
```

I will add these error throw tests

**Comment by ErikQQY** [comment]
@Azzaare I notice it is another commit causes the codecov failure, is there anything I can do to help solve the failure?

682D0015-1861-48A7-8B53-3EB6E1A58129

**Comment by Azzaare** [comment]
@ErikQQY it seems the content of the tests for `eratosthenes` for deleted. Can you put them back? Here is a link to the math test file on the main branch: https://github.com/TheAlgorithms/Julia/blob/ddb04b2f58c3bb52455ffdf87d56f59a836d4ec7/test/math.jl#L237
I can also do it if you are busy!

Thanks for coping with this long PR process :)

**Comment by ErikQQY** [comment]
No problem, I am working on it

File: src/TheAlgorithms.jl [hunk 1]
```diff
@@ -70,8 +70,15 @@ export sum_gp
 export surfarea_cube
 export surfarea_sphere
 export trapazoidal_area
-export eratosthenes
```
Comment (ErikQQY) [inline src/TheAlgorithms.jl:73]: I am confused, while in my fork, the eratosthenes algorithms is synchronous with this repo, but the pull request displays that I delete the algorithm export and algorithm include in **TheAlgorithms.jl** file

Comment (Azzaare) [reply src/TheAlgorithms.jl:74]: I moved it above to have it in alphabetical order. So it should not be missing

**Review by Azzaare — APPROVED** [review]

**Review by Azzaare — APPROVED** [review]

**Comment by ErikQQY** [comment]
All of the checks pass now

**Pull request merged**